.webpack/webpack.base.js

```diff
@@ -68,15 +66,7 @@ module.exports = (env, argv, { DIST_DIR }) => {
       // Show build progress
       new webpack.ProgressPlugin(),
       // Clear dist between builds
-      new CleanWebpackPlugin(),
-      new CopyPlugin({
-        patterns: [
-          {
-            from: '../../node_modules/cornerstone-wado-image-loader/dist/dynamic-import/',
-            to: DIST_DIR,
-          },
-        ],
-      }),
+      new CleanWebpackPlugin()
```

Comment:
So we are not using the dynamic import by default? Since we don't have the `from 'cornerstone-wardo-image-loader/dist/dynamic-import` anywhere in the code

Reply:
good point, I think we need the alias instead of just the copyPlugin. I'll fix that before we merge this.